libs/blocks/uar-results/uar-results.js

```diff
@@ -1,59 +1,94 @@
-/* eslint-disable no-await-in-loop */
 import { createTag } from '../../utils/utils.js';
 import { getMetadata, handleStyle } from '../section-metadata/section-metadata.js';
 
 async function loadFragments(el, experiences) {
   // eslint-disable-next-line no-restricted-syntax
   for (const href of experiences) {
+    /* eslint-disable no-await-in-loop */
     const a = createTag('a', { href });
     el.append(a);
     const { default: createFragment } = await import('../fragment/fragment.js');
     await createFragment(a);
   }
 }
 
-function redirectPage(quizUrl) {
+function redirectPage(quizUrl, debug, message) {
   const url = (quizUrl) ? quizUrl.text : 'https://adobe.com';
-  console.log(`Invalid Result, redirecting to: ${url}`);
+  window.lana.log(message);
+
+  if (debug === 'uar') {
+    // eslint-disable-next-line no-console
+    console.log(`${message}, redirecting to: ${url}`);
+  } else {
+    window.location = url;
+  }
+}
+
+function setAnalytics(hashValue, debug) {
```

Comment:
I am curious, the kind of setting up should be a common method for each project, right?

Reply:
Not really. We're the only block that is doing any kind of custom analytics of any kind.